src/pages/introduction/components/pic1.tsx

```diff
@@ -0,0 +1,66 @@
+import React,{FC} from 'react'
+import './styles/pic1.css'
+import {Box,Grid, GridItem, SimpleGrid, Stack, HStack, VStack, Image, Text, Flex, Heading } from '@chakra-ui/react';
+
+// import { Grid } from '@material-ui/core';
+const MyComponentOne: FC = () => {
+    return (
+      <div>
+        <div className='first-pic grid-container'>
+          <div className='grid-item-1'>
+            <Text>Hello 👋 Thanks for installing me. Let me get you started real quick.</Text>
+          </div>
+          <div className='grid-item-1'>
+            <img src="./assets/robot_3d.png"/>
+          </div>
+        </div>
+        
+        <div className='second-pic grid-container-2'>
```

Comment:
since we're using Tailwind for our styling needs, it's better to stick with it for consistency and maintainability purposes. I kindly ask you to switch to using Tailwind classes instead of plain CSS

Reply:
Will update it and inform you

Reply:
@Royal-lobster I am still having some college stuff, will finish the intro page within few days

Reply:
Sure ! that's fine